### ISSUE TITLE
Build docker images with buildx

### ISSUE THREAD

**Issue by muCruiser** [eikek/docspell#791]
I did build docspell with buildx for arm64 and basically it worked, allthough i could not get solr yet running.
i don't know your build-chain for docker, but perhaps you can try to change your build commands to

`buildx build --platform=linux/arm64/v8` + all the plattforms you want to build for

so we would have docker images for more plattforms available

**Comment by eikek** [comment]
Hi @muCruiser , thank you for this input. I didn't know about this extended build command. TBH the whole docker stuff is a bit of an annoyance for me :). I played quite often with the thought to remove it and hope that someone with more knowledge than me could pick it up. I don't use docker myself and I only have a limited knowledge from playing around with it, I think I especially miss all the tooling around. Each time I do a release the docker stuff holds me back, because it often doesn't work anymore and I have to dig into bash scripts and package-managers - it usually needs extra attention.

I think I need to reorganize the repo anyways (as suggested in https://github.com/eikek/docspell/issues/675#issuecomment-797690437). It would be great to provide images for different architectures, of course. Maybe this can be done in the same effort.

So TL;DR it will take some time to pick it up for me. If someone else wants to address it, I gladly give push rights and learn.

I'm also thinking about (for the short term) to go back to a simple build script that creates and pushes the images and remove the "build on dockerhub" pipeline.
